src/design-tokens/mixins.css

```diff
@@ -7,20 +7,24 @@
 }
 
 @define-mixin textLink {
+  /* TODO: inherit size */
   @mixin eds-theme-typography-body-text-md;
-  color: var(--eds-theme-color-link-foreground);
+  /* TODO: underline change on hover */
   text-decoration: underline;
+  color: var(--eds-theme-color-text-link-strong);
```

Comment:
The "strong" link tokens are purple whereas the default ones are gray. I need to verify that we're not using gray links in the platform; that might be the only variant we're not supporting that is being used.

Reply:
does that mean the future design direction is to have gray links by default? 🤔

Reply:
I'm not sure 🤔 I can't find any updated button/link designs, only the tokens page and some placeholder pages for buttons and links that I think are waiting on Sean's work. I wonder if we'll end up with two link variants, like `link-strong` and `link-default`?

Reply:
I did notice gray links in the presentation today.